src/methods.jl

```diff
@@ -111,8 +111,15 @@ for f in [identity, one, zero, *, +, -]
     @eval promote_symtype(::$(typeof(f)), T::Type{<:Number}) = T
 end
 
+# All of these need to be `Sym` to overwrite the function definitions of
+# the L78, that are made with the type `Sym <: Symbolic`
+#   @number_methods(Sym, term(f, a), term(f, a, b), skipbasics)
 promote_symtype(::typeof(Base.real), T::Type{<:Number}) = Real
-Base.real(s::Symbolic{<:Number}) = islike(s, Real) ? s : term(real, s)
+Base.real(s::Sym{<:Real}) = s
```

Comment:
why is this only `Sym` and not `Symbolic`?

Reply:
Oh I just read your comment. Well this seems like it should not be defined in L78, but should be defined outside of the loop there (but could be within `@number_methods` macro). So we should probably remove `real`, `conj` and `imag` from the list of one-argument functions.

Reply:
I'll move them and update the PR.

Reply:
Should I force push or add a commit?

Reply:
New commit is fine. Either is fine.